lib/plugins/aws/package/compile/events/msk/index.js

```diff
@@ -102,6 +108,10 @@ class AwsCompileMSKEvents {
             mskResource.Properties.BatchSize = batchSize;
           }
 
+          if (maximumBatchingWindowInSeconds) {
```

Comment:
> When this feature was implemented, I believe the maximumBatchingWindowInSeconds was not supported by msk - could you confirm that is the case?

It could be as recently as a few days ago: https://aws.amazon.com/blogs/compute/introducing-aws-lambda-batching-controls-for-message-broker-services/

Reply:
> If this is a bugfix, then I believe it should be applied for all other events that use event source mapping (stream, sqs for example). Could you confirm that it applies to all these types of events?

I will amend the PR accordingly with the other event types

Reply:
> If we confirm that we should validate it for all events mentioned in #2, then we should actually validate that maximumBatchingWindowInSeconds is satisfied and throw a meaninful error if that's not the case.

Interestingly, although AWS docs say "_must_" (this to me, implies it is mandatory), setting the batch size > 10  causes no failure when applying the underlying cloud formation template. Maybe AWS defaults to 1. Either way, happy to do as you suggest and make a validation error occur if the condition is not met.

Reply:
Thanks a lot for confirmation @rm-hull - I didn't catch that announcement before. As for the fixes, let's maybe start with just including the support for this new field and later we can evaluate if that additional validation is really needed. What do you think? Oh, after reading the article I've noticed that we should also support it for `activemq`, `kafka` and `rabbitmq` events. Would you like to add support for them as well? We can add them one by one in separate PRs

Reply:
Ok, so that means this PR is ok in its current form? .. and I will submit more PRs for the other event types 👍

Reply:
ooh ... I just noticed that `maximumBatchingWindow` is used for SQS (#8555).
I will change this PR to use the same naming (i.e.drop the "InSeconds" suffix from the serverless.yaml)